.prettierrc.json

```diff
@@ -2,21 +2,5 @@
   "semi": true,
   "trailingComma": "all",
   "printWidth": 120,
-  "overrides": [
-    {
-      "files": "*.test.ts",
-      "options": {
-        "semi": true
-      }
-    },
-    {
-      "files": [
-        "*.html",
-        "**/*.ts"
-      ],
-      "options": {
-        "tabWidth": 4
-      }
-    }
-  ]
+  "tabWidth": 4
```

Comment:
Note:
This is the same as was before.
- `*.test.ts` already have `semi: true` from the parent
- all files for prettier already have tabsWidth: 4